### DISCUSSION TITLE
Ability to specify icons for QDate

### DISCUSSION THREAD

**Discussion by robert-hanuschke** [quasarframework/quasar#13674] [Ideas / Proposals] [answered]
I decided to not include a whole icon font but import the icons I need directly.
That works fine for most Quasar components, I'm hitting a wall at QDate though.
It seems there is no possibility to specify icons for the forward, backward and today button functionality. This means I'm stuck with the string representations showing up way too big and overlapping, making the currently shown month and year illegible.
Would love to see the possibility to specify icons as is available for other components.

**Comment by alexrififi** [comment, marked as answer]
See this https://quasar.dev/options/quasar-icon-sets#changing-a-specific-icon-dynamically

```ts
$q.iconSet.datetime.xxx = yourIcon // xxx = arrowLeft | arrowRight | now | today
```

**Reply by robert-hanuschke** [reply]
that works for me without requiring a parameter for the icons on QDate directly, thank you!
